src/components/TokenClaimDialog.tsx

```diff
@@ -72,6 +93,7 @@ export default function TokenClaimDialog({
   const formattedTotalRetroDrop = commify(
     formatBNToString(rewardBalances.retroactiveTotal, 18, 2),
   )
+
   const [allPoolsWithRewards, poolsWithUserRewards] = useMemo(() => {
```

Comment:
adapt this to work with gauges, or delete it. We do like that it's sorted by reward balance though.

Reply:
also change `allPoolsWithRewards` on line 193 to reference your gauge data https://github.com/saddle-finance/saddle-frontend/pull/1015/files#diff-46b08b90f09d1cc35a7f88fb50620b10f86ffda551aa713fb5baa65701c95e22R193